MAINTAINERS.yml

```diff
@@ -2343,7 +2343,6 @@ West:
   maintainers:
     - ifyall
   collaborators:
-    - nandojve
```

Comment:
any reason `@npal-cy` isn't listed as collaborator?  Feels like this file needs updating for all this.

Reply:
added.